Chinese/Casey/Casey.md

```diff
@@ -1,3 +1,5 @@
 Casey现在为udacity工作，自动合并GitHub的pull request。
 Casey is a help bot.
-Casey won't accept a pull request that a fix typo.
\ No newline at end of file
+Casey won't accept a pull request that a fix typo.
+直到有一天Casey收到来自转行开始学编程的YouTuber的pull request.
+他就是[CaseyNeistat](Chinese/Casey/neistat/neistat.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.